packages/server-core/src/media/static-resource/static-resource.hooks.ts

```diff
@@ -74,7 +97,7 @@ export default {
     remove: [
       authenticate(),
       // iff(isProvider('external'), verifyScope('admin', 'admin') as any),
-      attachOwnerIdInQuery('userId')
```

Comment:
Should we send userId in params? This causes feathers to not find any items with the associated resource id when an admin is removing a non owned static-resource because of user id mismatch